_data/software/openshot.yml

```diff
@@ -0,0 +1,36 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      OpenShot Video Editor is a free and open-source video editor for FreeBSD,
+      Linux, Haiku, macOS, and Windows.
+    fr: >-
+      OpenShot Video Editor est un éditeur vidéo gratuit et open-source pour
+      FreeBSD, Linux, Haiku, macOS et Windows.
+category: '43232100'
+homepageURL:
+  en: 'https://www.openshot.org/'
+  fr: 'https://www.openshot.org/fr/'
+licences:
+  - URL:
+      en: 'https://github.com/OpenShot/openshot-qt/blob/develop/COPYING'
+      fr: 'https://github.com/OpenShot/openshot-qt/blob/develop/COPYING'
+    spdxID: GPL-3.0-only
```

Comment:
Looks like it might be an "or later" license https://github.com/OpenShot/openshot-qt#copyright